src/libs/actions/Report.js

```diff
@@ -773,28 +788,44 @@ function fetchAllReports(
                 return fetchChatReportsByIDs(reportIDs);
             }
 
-            return fetchOrCreateChatReport([currentUserEmail, 'concierge@expensify.com'], false)
-                .then((createdReportID) => {
-                    reportIDs = [createdReportID];
-                });
+            return fetchOrCreateChatReport([currentUserEmail, 'concierge@expensify.com'], false);
```

Comment:
Instead of modifying `fetchOrCreateChatReport` you could just return the array like:

```js
return fetchOrCreateChatReport([currentUserEmail, 'concierge@expensify.com'], false)
  .then(reportId => [reportId]); 
```

IMO it's better than having to justify something fishy with a comment

```js
            // We are returning an array with the reportID here since fetchAllReports calls this method or
            // fetchChatReportsByIDs which returns an array of reportIDs.
            return [data.reportID];
```
- this just discloses a coupling to the `fetchAllReports` usage

Reply:
Thanks for your notes here. I see no major problem with doing it either way.